platform-modules/pom.xml

```diff
@@ -19,7 +19,7 @@
 		<version.compiler.plugin>2.3.1</version.compiler.plugin>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<maven.compiler.source>1.8</maven.compiler.source>
-		<cloud.store.version>1.2.8</cloud.store.version>
+		<cloud.store.version>1.4.4</cloud.store.version>
```

Comment:
@ddevadat - OCI supported with 1.4.6 right?

Reply:
corrected the version. Do i need to update the version in spike module also? i am not sure if its in use